src/plugins/vctlGit.nim

```diff
@@ -13,49 +13,51 @@ import pkg/[zippy, zippy/inflate]
 import ".."/[config, plugin_api]
 
 const
-  eBadGitConf     = "Git configuration file is invalid"
-  fanoutTable     = 8
-  fanoutSize      = (256 * 4)
-  fNameHead       = "HEAD"
-  fNameConfig     = "config"
-  highBit32       = uint64(0x80000000)
-  gpgSignStart    = "-----BEGIN PGP SIGNATURE-----"
-  gpgSignEnd      = "-----END PGP SIGNATURE-----"
-  ghRef           = "ref:"
-  ghBranch        = "branch"
-  ghRemote        = "remote"
-  ghUrl           = "url"
-  ghOrigin        = "origin"
-  ghLocal         = "local"
-  gitObject       = "object"
-  gitAuthor       = "author"
-  gitCommitter    = "committer"
-  gitTag          = "tag"
-  gitSign         = "gpgsig"
-  gitTagger       = "tagger"
-  gitIdxAll       = "*.idx"
-  gitIdxExt       = ".idx"
-  gitIdxHeader    = "\xff\x74\x4f\x63\x00\x00\x00\x02"
-  gitObjects      = "objects"
-  gitPack         = gitObjects.joinPath("pack")
-  gitPackExt      = ".pack"
-  gitTimeFmt      = "ddd MMM dd HH:mm:ss YYYY"
-  gitObjCommit    = 1
-  gitHeaderType   = "$type"
-  keyVcsDir       = "VCS_DIR_WHEN_CHALKED"
-  keyOrigin       = "ORIGIN_URI"
-  keyCommit       = "COMMIT_ID"
-  keyCommitSigned = "COMMIT_SIGNED"
-  keySigned       = "COMMIT_SIGNED"
-  keyBranch       = "BRANCH"
-  keyAuthor       = "AUTHOR"
-  keyAuthorDate   = "DATE_AUTHORED"
-  keyCommitter    = "COMMITTER"
-  keyCommitDate   = "DATE_COMMITTED"
-  keyLatestTag    = "TAG"
-  keyTagSigned    = "TAG_SIGNED"
-  keyTagger       = "TAGGER"
-  keyTaggedDate   = "DATE_TAGGED"
+  eBadGitConf      = "Git configuration file is invalid"
+  fanoutTable      = 8
+  fanoutSize       = (256 * 4)
+  fNameHead        = "HEAD"
+  fNameConfig      = "config"
+  highBit32        = uint64(0x80000000)
+  gpgSignStart     = "-----BEGIN PGP SIGNATURE-----"
+  gpgSignEnd       = "-----END PGP SIGNATURE-----"
+  ghRef            = "ref:"
+  ghBranch         = "branch"
+  ghRemote         = "remote"
+  ghUrl            = "url"
+  ghOrigin         = "origin"
+  ghLocal          = "local"
+  gitObject        = "object"
+  gitAuthor        = "author"
+  gitCommitter     = "committer"
+  gitCommitMessage = "commitMessage"
```

Comment:
these are the fields meant to describe git object file which applies to both commits and tags therefore `commitMessage` is misleading. we can do simple `gitMessage`